README.md

```diff
@@ -37,6 +37,13 @@ the `vets-api` directory, run `./bin/setup-osx && source ~/.bash_profile && cd .
   - `brew install redis`
   - Follow post-install instructions to enable Redis on startup. Otherwise,
     launch it manually with `brew services start redis`.
+1. Install ClamAV
+  - `brew install clamav`
+  - `cd` into the directory listed by the post-install instructions (and replace '${post_install_dir}' with it below), then:
+    - `touch clamd.sock`
+    - `echo "LocalSocket ${post_install_dir}" > clamd.conf` 
+    - `echo "DatabaseMirror database.clamav.net" > freshclam.conf`
+    - `freshclam -v`
```

Comment:
I attempted the steps and failed to install ClamAV properly.  Either I'm a dumb user or the steps need.  Probably both.

I've since completed the install but it involved renaming a `freshclam.conf.sample` to `freshclam.conf` which the steps don't say anything about.  I think I incorrectly assumed what directory `${post_install_dir}` meant.